seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/sink/client/ClickhouseProxy.java

```diff
@@ -141,7 +141,7 @@ public Map<String, String> getClickhouseTableSchema(ClickHouseRequest<?> request
      */
     public List<Shard> getClusterShardList(ClickHouseRequest<?> connection, String clusterName,
                                            String database, int port, String username, String password) {
-        String sql = "select shard_num,shard_weight,replica_num,host_name,host_address,port from system.clusters where cluster = '" + clusterName + "'";
+        String sql = "select shard_num,shard_weight,replica_num,host_name,host_address,port from system.clusters where cluster = '" + clusterName + "'" + " and replica_num=1";
```

Comment:
As the document say we only support the distributed table with internal_replication equals to true,  so when we split data , we only need to split data to one replica of shard, and let internal replication to copy to other replicas. But I am hesitating whether to add a configuration like internal_replication to let user decide whether write data to all replica or just one replica of a shard